code/client/clrcore/External/AiBlip.cs

```diff
@@ -0,0 +1,177 @@
+using System;
+
+#if MONO_V2
+using CitizenFX.Core;
+using API = CitizenFX.FiveM.Native.Natives;
+
+namespace CitizenFX.FiveM
+#else
+using CitizenFX.Core.Native;
+
+namespace CitizenFX.Core
+#endif
+{
+	public sealed class AiBlip : PoolObject, IEquatable<AiBlip>
```

Comment:
`AIBlip`, convention is to turn acronyms into Pascal case when we have more than 2 characters. Natives are like that purely because the tools indiscriminately apply PascalCase.

Also why seal it?